seatunnel-connectors-v2/connector-slack/src/main/java/org/apache/seatunnel/connectors/seatunnel/config/SlackConfig.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.seatunnel.connectors.seatunnel.config;
+
+import org.apache.seatunnel.shade.com.typesafe.config.Config;
+
+import lombok.Data;
+import lombok.NonNull;
+
+@Data
+public class SlackConfig {
+
+    public static final String WEBHOOKS_URL = "webhooks_url";
+    public static final String OAUTH_TOKEN = "oauth_token";
+    public static final String SLACK_CHANNEL = "slack_channel";
+    private String webHooksUrl;
+    private String oauthToken;
+    private String slackChannel;
+
+    public SlackConfig(@NonNull Config pluginConfig) {
+        if (pluginConfig.hasPath(WEBHOOKS_URL)) {
```

Comment:
These three parameters have already been checked in the prepare method, and this place does not need to check again.

Reply:
I have removed it.PTAL